pretext/Functions/ChapterAssessment.ptx

```diff
@@ -144,4 +144,89 @@
       </input>
     </program>
   </exercise>
+  <exercise label="ac4_15_6">
+    <statement>
+      <p>You will need to write three functions for this problem.</p>
+      <ul>
+        <li>
+          <p> The first function, <c>average</c>
+          that takes in any three numbers and returns the average.
+          </p>
+        </li>
+        <li>
+          <p> The second function
+          called <c>diffSquare</c> that takes one number and the average of the three numbers, subtracts those numbers, and then returns the square of this difference.
+          </p>
+        </li>
+        <li>
+          <p>
+            A third function called <c>stdev</c> which is passed the three numbers and then:
+            </p>
+            <ol>
+              <li>
+                <p>
+                  Calls on <c>average</c> to find the averaage of those three values, and assign the resulting average to a local variable.
+                </p>
+              </li>
+              <li>
+                <p>
+                  Calls on <c>diffSquare</c> three times, passing each of the three numbers in turn and the local variable average, assigning the resulting squared differences to three
+                  local variables.
+                </p>
+              </li>
+              <li>
+                <p>
+                  Calls on <c>average</c> again, this time passing in the three squared differences, and assigning this new average to a variable called <c>variance</c>.
+                </p>
+              </li>
+            </ol>
+        </li>
+        <li>
+          <p>
+            Finally, <c>stdev</c> finds the square root of <c>variance</c> as the standard deviation
+          </p>
+        </li>
+      </ul>
+      <p> Do not worry about rounding the decimals.</p>
+    </statement>
+    <program xml:id="ac4_15_6_editor" interactive="activecode" language="python">
+      <input>
+      """
+      Using these values: 10, 13, 15,
+      you should get:
+      - an average of 12.66666667.
+      - these squared differences are: 7.1111111 0.11111111, 5.4444444
+      - which sum to 12.66666667 and average to be 4.2222222
+      - the square root of which is: 2.054804667
+        """"
+
+      def average( ,  ,  ): 
+
+
+
+
+        return mean
```

Comment:
The rest of the book seems to be 4 spaces indentation, so to be consistent I changed this example as well.

Reply:
Actually, it's inconsistent, whoops.